bluemira/geometry/tools.py

```diff
@@ -871,6 +872,22 @@ def save_as_STEP(shapes, filename="test", scale=1):
     cadapi.save_as_STEP(freecad_shapes, filename, scale)
 
 
+def save_cad(components, filename="test", scale=1):
```

Comment:
Is there a requirement to have a default file name here? Usually, '`save`' functions require a path to be specified. I foresee some accidental file overwrites with a default.

Reply:
Year fair enough, it was really just copied from `save_as_STEP` i'll remove the default from both